src/create-theme-listener.js

```diff
@@ -0,0 +1,27 @@
+const PropTypes = require('prop-types');
+const channel = require('./channel');
+
+function createThemeListener(CHANNEL = channel) {
```

Comment:
its minor implementation detail, and lets discuss to change it later in separate issue